### PR TITLE
Notebook fix

### PR DESCRIPTION
**Pull request by pshivraj**
This small PR fixes output folder creating in the ```Synthetic-Data-Creation-for-Retail-Environments.ipynb``` notebook.

```
output_dir = "data/"
if not os.path.exists(output_dir ):
    os.mkdir(output_dir )
```

### CODE REVIEW

**Comment by havanagrawal** [comment]
Instead of handling this in the notebook, just replace this line [here](https://github.com/havanagrawal/clomask/blob/master/data-acquisition/data-synthesizer/src/data_synthesizer/data_synthesizer.py#L229) with 
```python
os.makedirs(save_path)
```

**Comment by pshivraj** [comment]
> Instead of handling this in the notebook, just replace this line [here](https://github.com/havanagrawal/clomask/blob/master/data-acquisition/data-synthesizer/src/data_synthesizer/data_synthesizer.py#L229) with
> 
> ```python
> os.makedirs(save_path)
> ```
Thanks this makes much more sense.

This has been fixed in the latest commit.

File: data-acquisition/data-synthesizer/src/data_synthesizer/data_synthesizer.py [hunk 1]
```diff
@@ -226,7 +226,7 @@ def generate_synthetic_dataset(
         timestamp = time.strftime("%Y_%m_%d_%H_%M_%S", time.gmtime())
         dataset_name = "synth_data_{}".format(timestamp)
         save_path = os.path.join(output_dir, dataset_name)
-        os.mkdir(save_path)
+        os.mkdirs(save_path)
```
Comment (havanagrawal) [inline data-acquisition/data-synthesizer/src/data_synthesizer/data_synthesizer.py]: [`os.makedirs`](https://docs.python.org/3/library/os.html#os.makedirs)

**Review by havanagrawal — APPROVED** [review]
👍

**Pull request merged**